portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,69 @@
 #content {
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 600px;
 }
-
-#greeting-container {
+#image {
+    margin-left: auto;
+    margin-right:auto;
+    margin-top: auto;
+    width: 128px;
+}
+#fact-container {
   margin-top: 20px;
 }
+
+body {
+    background-color: lightblue;
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, 
+    Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
```

Comment:
Because this is a continuation of the `font-family` rule above, indent this 4 extra spaces.